provision-runners/README.md

```diff
@@ -0,0 +1,44 @@
+# Provision Runners
+
+Use the `provision-runners` action in the following way: 
+```yaml
+        - uses: ./provision-runners
+          with: 
+            aws_access_key_id: #'Use a secret - this account needs "ec2-admin" permissions'
+            aws_secret_access_key: #'Use a secret - this account needs "ec2-admin" permissions'
+            github_repo_token: #"Use a secret. Token used to register a github action runner. If D6E Automaton token make sure it is added to the repo with Admin powers."
+            region: #'aws region for the runner. Default us-east-1'
+            ami_id: #"ami ID that will run on the mac2 instance. Must be associated with the Host Resource Group license."
```

Comment:
Updated - it only needs to be associated with the HRG license if it will be running on the HRG (mac instances). All instances need to identify an ami id.